### PR TITLE
changes backgrounds to be more generic, changes fate descriptions

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

changes backgrounds to not be ultra specific to eris lore

also changes some adding 35 stats total lol

changes fate descriptions, makes some more clear & corrects typos

## Why It's Good For The Game

clarity good, eris lore man not in eclipse

## Changelog
:cl:
tweak: changed background descriptions
balance: changed background stats
tweak: changed fate descriptions
:cl:

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
merging #283 with this since it's literally just the same thing basically

File: code/datums/setup_option/backgrounds/fate.dm [hunk 1]
```diff
@@ -22,57 +22,56 @@
...
 	stat_modifiers = list(STAT_COG = -10)
 	perks = list(PERK_ALCOHOLIC)
 
 /datum/category_item/setup_option/background/fate/noble
 	name = "Noble"
-	desc = "At some point of the past your family played important part in human history. \
-			Those were captains and bridge crew of the colony ships, a CEO of first corporations, an admirals and generals. \
-			All of them decided to entrench their accomplishments by creating noble leaninges, and here you are, a lost soul of high origin. \
-			What does it makes you?"
+	desc = "You are of a prestigious family or lineage. Maybe you're self-made. Maybe you're the heir to a fortune..\
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/fate.dm:64]: Your period key double-struck here.

File: code/datums/setup_option/backgrounds/fate.dm [hunk 2]
```diff
@@ -22,57 +22,56 @@
...
+			Either way, you, like everyone else, are stuck aboard the NEV Northern Light. This is not the luxury you are used to. \
+			You're far more likely to be targeted for abduction by...unsavory types. \
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/fate.dm:66]: Space between ellipsis and next word please

(`by... unsavory`)

File: code/datums/setup_option/backgrounds/fate.dm [hunk 3]
```diff
@@ -22,57 +22,56 @@
...
+			Stay safe, star-child."
 
 	perks = list(PERK_NOBLE)
 
 /datum/category_item/setup_option/background/fate/rat
 	name = "Rat"
-	desc = "In your life you decided to feast upon what’s not yours, be that thief, raiding, or scavenging and exploring. It’s all the same no matter how you name it, after all. \
-			You know the ways of infiltrating, salvaging, and getting away with the loot."
+	desc = "You're used to acquiring things by...repurposing what others have. \
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/fate.dm:73]: Space between ellipsis and next word please

(`by... repurposing`)

File: code/datums/setup_option/backgrounds/origin.dm [hunk 4]
```diff
@@ -7,176 +7,144 @@
...
-	name = "Predstraza"
-	desc = "The product of a widely held idea back on Earth that if one were to bring all of the Balkan populations on one colony ship, they would be forced to put aside their differences and work together to survive. \
-			Unfortunately, that idea failed miserably and is still failing. \
-			A jungle death world, full of hostile fauna and mired in endless armed conflict. \
-			The Serbian government occupied all of Predstraza�s space ports and controls most of the civilized regions. \
-			Located far from Hanza or NT territories, it�s a place favored for business by pirates, smugglers and all manner of outlaw."
+	name = "Thug"
+	desc = "You're an unsavory sort, aren't you? \
+			Maybe you're a criminal, maybe you just got into a lot of fights. Either way, you know how to swing your weight around and how to not get caught off-guard. \
+			...all those blows to your head may have knocked a few things loose, though..."
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/origin.dm:28]: Capitalize first word of sentence

File: code/datums/setup_option/backgrounds/origin.dm [hunk 5]
```diff
@@ -7,176 +7,144 @@
...
 	)
 
 
 /datum/category_item/setup_option/background/origin/sich_prime
-	name = "Sich Prime"
-	desc = "A Ukrainian colony that was a major industrial center during the Corporation Wars, during which they made the fatal mistake of siding with Nanotrasen. \
-			Bombed and poisoned, this planet suffered the harshest toll of the war, both from population loss and ecological devastation. \
-			While ostensibly under the direct control of Hanza, its local military is extremely disloyal and known to support the Founders - a revanchist international group that seeks the return of planetary government's national control over space."
+	name = "Vagrant"
+	desc = "Miscreant. You're used to having to do what it takes to survive, patching up your own wounds to survive, and being...familiar, with how to get through locks and
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/origin.dm:42]: being... familiar

File: code/datums/setup_option/backgrounds/origin.dm [hunk 6]
```diff
@@ -7,176 +7,144 @@
...
-	name = "HMSS \"Destined\""
-	desc = "A British colony ship that was one of those who failed to locate a habitable world before exhausting its fuel supply; however, unlike the others, the crew managed to survive by turning their ship into the largest station in Hanza controlled space. \
-			Plated in rusty metal, with high costs of living, a permanent space roach infestation and no natural ecology to speak of, the \"Destined\" can be aptly described as an industrial hell. \
-			While The Corporation War was rather merciful to it, and the station remained neutral throughout most of the conflict, in the end, it succumbed to Syndicate occupation. \
-			The \"Destined\" is widely known for its anachronistic Monarchical government and system of noble peerage, with most successful nobles controlling smaller stations near it and others bent on quelling the chaos within the Colony proper. \
-			They are also members of Asters Guild, and this whole station is considered major Guild territory."
+	name = "Crewman"
+	desc = "You're not unfamiliar withh working on spaceships. Maybe you've been working for NanoTrasen for some time, or maybe you used to be part of the navy.\
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/origin.dm:87]: typo'd "with"

Also, can you edit 'the navy' to be 'some navy' or something similar? There's many planets, there's many cultures, there's many navies.

File: code/datums/setup_option/backgrounds/origin.dm [hunk 7]
```diff
@@ -7,176 +7,144 @@
...
+			 Either way, you know the- general- ins and outs of working on a ship."
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/origin.dm:88]: "know the general ins-and-outs of working"

File: code/datums/setup_option/backgrounds/origin.dm [hunk 8]
```diff
@@ -7,176 +7,144 @@
...
 /datum/category_item/setup_option/background/origin/nss_forecaster
-	name = "NSS \"Forecaster\""
-	desc = "Designed to serve as large, FTL capable mining platform by the first days of NanoTrasen. \
-			And it was used for that, until stars started coming back from Null Space. \
-			In order to salvage those wonders first, NanoTrasen has sent this platform, reworked to serve as a local forward operating base, and renamed it to Central Command. \
-			The war broke out, and without any support from the mainland, after relentless attacks from the Syndicate, and the destruction of many stations under CentComm�s control, such as NSS 13 \"Exodus\", they surrendered to Syndicate. \
-			Now it�s an independent station under Ironhammer control, that oversees the spread of Null Space artifacts, or at least is trying to. \
-			It�s a place for grand deals to be made, friends to be sold, a place where people run from law and boring life,for a fresh start in Null Space. \
-			And - in most cases - die a horrible death in the end."
+	name = "Nomad	"
```
Comment (EvilJackCarver) [inline code/datums/setup_option/backgrounds/origin.dm:144]: What's with the tab between the name and closequote here?

Comment (bearrrrrrrr) [reply code/datums/setup_option/backgrounds/origin.dm:144]: I have never made typos in my life, ever

**Comment by bearrrrrrrr** [comment]
Should be good now

**Comment by EvilJackCarver** [comment]
Just the one thing missed, I've unresolved it for now.

**Comment by bearrrrrrrr** [comment]
That was a purposeful choice but will-do, chief.

**Pull request merged**